### ISSUE TITLE
Ubuntu 22.04 user id

### ISSUE THREAD

**Issue by jmthomas** [OpenC3/cosmos#576]
During a recent training a user on Ubuntu 22.04 had issues in the scripts because his user id was not set. We set user and group in the scripts as follows:
```
export OPENC3_USER_ID=`id -u`
export OPENC3_GROUP_ID=`id -g`
```
Verify this still works in Ubuntu 22.04.

**Comment by jmthomas** [comment]
Also see if we can hide `docker: compose is not a docker command` on Linux

**Comment by jmthomas** [comment]
Fixed by #815
